tests/CsvFileTest.php

```diff
@@ -195,6 +196,77 @@ public function testWrite()
         foreach ($rows as $row) {
             $csvFile->writeRow($row);
         }
+        $data = file_get_contents($fileName);
+        self::assertEquals(
+            "\"col1\",\"col2\"\n" .
+            "\"line without enclosure\",\"second column\"\n".
+            "\"enclosure \"\" in column\",\"hello \\\"\n" .
+            "\"line with enclosure\",\"second column\"\n" .
+            "\"column with enclosure \"\", and comma inside text\",\"second column enclosure in text \"\"\"\n" .
+            "\"columns with\nnew line\",\"columns with\ttab\"\n" .
+            "\"column with \\n \\t \\\\\",\"second col\"\n",
+            $data
+        );
```

Comment:
Ještě bych rovnou přidal unlink sem, ať to nezůstává zbytečně pro happy path na filesystému.